Disable test that use package installed in ${ROOT}

https://bugs.gentoo.org/show_bug.cgi?id=429668

Patch written by Ian Delaney <johneed@hotmail.com>
--- apptools/persistence/tests/check_version_registry.py
+++ apptools/persistence/tests/check_version_registry.py
@@ -39,6 +39,7 @@
 
 
 class TestVersionRegistry(unittest.TestCase):
+    @unittest.skip("This tests uses installed version")
     def test_get_version(self):
         """Test the get_version function."""
         c = Classic()
